docs/client-options.md

```diff
@@ -93,6 +93,7 @@ third party clients.
 - [CertLE](https://github.com/skoerfgen/CertLE)
 - [AcmePHP](https://github.com/acmephp/acmephp)
 - [LE Manager](https://github.com/analogic/lemanager)
+
```

Comment:
I think this is a spurious diff - can you please remove it?

Reply:
Seems like the online editor is playing some strange games with me. Please cancel this pull request while I create a new one with normal editor.